### PR TITLE
drawer on /p/:projectId/f/:featureId

### LINKED ISSUE #9: [Map] Show feature details in side panel
Main user journey:
1. Click place marker on map
2. Right-side drawer opens, displacing entire UI (similar to side panel in Gmail or Docs)
3. Heading w/feature type name appears and list of completed forms (Records), each shown as a separate card in reverse chronological order.
4. Click "x" in top-right corner of panel to close.

### PR DESCRIPTION
**Pull request by dhvogel**
closes #9

### CODE REVIEW

**Comment by dhvogel** [comment]
@gino-m this is ready for review

**Comment by dhvogel** [comment]
for reference, this is what it looks like on my local. the underlying data structure is what you posted in #9

File: web/src/components/gnd-feature-details/index.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
```
Comment (gino-m) [inline web/src/components/gnd-feature-details/index.js]: Copyright 2019

File: web/src/components/gnd-feature-details/index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  paper: {
+    top: 62,
+    paddingTop: 20,
+    width: 300,
+  },
+  title: {
+    fontWeight: 'bold',
+  },
+  date: {
+    marginTop: -20,
```
Comment (gino-m) [inline web/src/components/gnd-feature-details/index.js]: Is there any way to do the same w/o using negative margins? They get messy to manage very fast..

File: web/src/components/gnd-feature-details/index.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        pullRight={true}
+        classes={{paper: classes.paper}}
+      >
+        {recordsList}
+      </Drawer>
+    );
+  }
+}
+
+// start of code change
```
Comment (gino-m) [inline web/src/components/gnd-feature-details/index.js]: Stray comment?

**Comment by dhvogel** [comment]
done

**Pull request merged**